schema/container.go

```diff
@@ -26,6 +26,9 @@ func (cm *ContainerRuntimeManifest) UnmarshalJSON(data []byte) error {
 	if err != nil {
 		return err
 	}
+	if c.Annotations == nil {
```

Comment:
You're right, the value is questionable. My validation code was using reflect.DeepEquals and failing as it was comparing nil and empty. But I mistakenly thought there's also a problem with calling annotations.Set() when the underlying slice is nil -- there's not.

I can revert this and easily work around this in the validator code.